app/javascript/packages/address-search/components/full-address-search.tsx

```diff
@@ -6,6 +6,7 @@ import type { LocationQuery, FormattedLocation } from '@18f/identity-address-sea
 import FullAddressSearchInput from './full-address-search-input';
 
```

Comment:
I like that you shortened file name. Now component name is the same as the file name.